app/services/db/monthly_sp_auth_count/total_monthly_auth_counts_within_iaa_window.rb

```diff
@@ -125,7 +101,7 @@ def partial_month_subqueries(issuer:, partial_months:)
             , COUNT(sp_return_logs.id) AS auth_count
             FROM sp_return_logs
             WHERE
-                  sp_return_logs.requested_at::date BETWEEN %{range_start} AND %{range_end}
+                  sp_return_logs.requested_at BETWEEN %{range_start} AND %{range_end}
```

Comment:
so I think that this change may mean these are essentially unindex queries now 😬 , this is our partial index:

https://github.com/18F/identity-idp/blob/c62f569ba319641f6e69f52c2e9b6064a9a21a27/db/schema.rb#L588

cc @mitchellhenke @stevegsa @jmhooper if you have any thoughts on if I should try to add a "plain" index on `(requested_at, issuer)`? (and it's a huge table so I know it would fail during a normal deploy)

Reply:
Yes we need requested_at and issuer does nothing it can be dropped.

Reply:
for the combined billing reports, we do break things out by issuer by month, so I think it does help for those?

Reply:
update, thanks to some investigation help from @mitchellhenke, the `::date` truncation does work like we expect so I undid this timestamp change part in c5b005ef06c4fa9c97c91dc480decf3d7e2061d4